core/server/worker/src/test/java/alluxio/worker/block/MonoBlockStoreCommitBlockTest.java

```
@@ -0,0 +1,142 @@
+        File tempFolder = mTestFolder.newFolder();
+        TieredBlockStoreTestUtils.setupDefaultConf(tempFolder.getAbsolutePath());
+
+        mockedBlockMasterClientPool = mock(BlockMasterClientPool.class);
+        mockedBlockMasterClient = mock(BlockMasterClient.class);
+        when(mockedBlockMasterClientPool.acquire()).thenReturn(mockedBlockMasterClient);
+        doNothing().when(mockedBlockMasterClientPool).release(any());
+        mBlockLockManager = new BlockLockManager();
+        mBlockMetadataManager = BlockMetadataManager.createBlockMetadataManager();
+
+        mTestDir1 = mBlockMetadataManager.getTier(FIRST_TIER_ALIAS).getDir(0);
+
+        mListener = mock(BlockStoreEventListener.class);
+        doAnswer((i) -> {
+            assertEquals(2L, i.getArguments()[0]);
+            return 0;
+        }).when(mListener).onCommitBlockToLocal(anyLong(), any(BlockStoreLocation.class));
+        doAnswer((i) -> {
+            assertEquals(2L, i.getArguments()[0]);
+            return 0;
+        }).when(mListener).onCommitBlockToMaster(anyLong(), any(BlockStoreLocation.class));
+    }
+
+    @Test
+    public void CommitLocalandCommitMasterBothSuccess() {
+        mTieredBlockStore = new TieredBlockStore(mBlockMetadataManager, mBlockLockManager);
+
+        prepareBlockStore();
+
+        mMonoBlockStore.commitBlock(sessionId, blockId, false);
+
+        verify(mListener).onCommitBlockToLocal(anyLong(), any(BlockStoreLocation.class));
+        verify(mListener).onCommitBlockToMaster(anyLong(), any(BlockStoreLocation.class));
+    }
+
+    @Test
+    public void CommitLocalSuccessandCommitMasterFail() {
+        try {
+            doAnswer((i) -> {
+                throw new AlluxioStatusException(Status.UNAVAILABLE);
+            }).when(mockedBlockMasterClient).commitBlock(anyLong(), anyLong(), anyString(), anyString(), anyLong(), anyLong());
+        } catch (Exception e) {
+            System.out.println(e);
+        }
```

do not suppress exception